cmd/podman/run.go

```diff
@@ -38,6 +39,7 @@ func init() {
 	flags.SetInterspersed(false)
 	flags.SetNormalizeFunc(aliasFlags)
 	flags.Bool("sig-proxy", true, "Proxy received signals to the process")
+	flags.Bool("rmi", false, "Remove pulled image unless used by other containers")
```

Comment:
"pulled" is not accurate as the image could already be in the storage. I suggest to: "Remove container image unless...."

Reply:
concur, but I'd go with "the container image" or perhaps just "the image" for brevity.